spec/domains/scores/city_scores_spec.rb

```diff
@@ -87,7 +97,7 @@
 
   describe "caching" do
     it "creates cache records on first call" do
-      expect { described_class.get }.to change { Cache::CityScore.count }.from(0).to(3)
+      expect { described_class.get }.to(change { Cache::CityScore.count }.from(0).to(3))
```

Comment:
Makes me want to hit Rubocop over the head with a shovel until there's not a single line of code left.